_posts/2021/2021-03-19-the-x-factor.md

```diff
@@ -0,0 +1,59 @@
+---
+title: The X factor or how I learned to stop worrying and love the backend
+layout: post
+date: 2021-03-19
+header_image: public/micronaut-header.jpeg
+header_position: center
+header_overlay: true
+category: on-the-job
+tags: ["agile", "company", "team"]
+authors: ["Paolo"]
+about_authors: ["ppriotto"]
+---
+
+It was autumn 2020. I had been at ePages for almost six years, and always in the same frontend team. You could say I was ready for some change. So when our scrum masters started asking around who could imagine joining a new, cross functional team, I was immediately up for it. Four months later, I want to recapitulate the journey since then.
+
+## Done actually
+
+Haven't we done this before? Yes and no. There have always been cross functional teams responsible for certain areas of the ePages products. As far as I can tell, they're successful and satisfied with their team structure. However, the biggest portion of the current product line has been developed following an "API first" approach, with a big core frontend team doing almost all the UI work. This led to cross-team dependencies on tasks, unavoidable handovers, and an "us vs. them" attitude. We'll take a deeper look at these issues in the following paragraphs.
```

Comment:
maybe "new product line" , instead of "current product line" - since NOW is more visible on the markets so far and ePages6 Unity is working x-functional